src/cephadm/cephadmlib/daemons/smb.py

```diff
@@ -201,6 +267,77 @@ def args(self) -> List[str]:
         return super().args() + ['update-config', '--watch']
 
 
+class CTDBMigrateInitContainer(SambaContainerCommon):
+    def name(self) -> str:
+        return 'ctdbMigrate'
+
+    def args(self) -> List[str]:
+        # TODO: not only do we want to only do this on node 0, we only
```

Comment:
Might be worth opening a tracker for now to have introduce migration status tracking?

Reply:
sure